### ISSUE TITLE
More Compile Errors Yay!!

### ISSUE THREAD

**Issue by CanadianJeff** [libretro/RetroArch#2472]
```
CC net_http_special.c
CC tasks/task_http.c
CC netplay.c
CC cheevos.c
CC libretro-common/utils/md5.c
LD retroarch
/usr/bin/ld: obj-unix/gfx/common/drm_common.o: undefined reference to symbol 'drmModeFreeCrtc'
//usr/lib/x86_64-linux-gnu/libdrm.so.2: error adding symbols: DSO missing from command line
collect2: error: ld returned 1 exit status
Makefile:119: recipe for target 'retroarch' failed
make: *** [retroarch] Error 1
CC git_version.c
LD retroarch
/usr/bin/ld: obj-unix/gfx/common/drm_common.o: undefined reference to symbol 'drmModeFreeCrtc'
//usr/lib/x86_64-linux-gnu/libdrm.so.2: error adding symbols: DSO missing from command line
collect2: error: ld returned 1 exit status
Makefile:119: recipe for target 'retroarch' failed
make: *** [retroarch] Error 1
```

**Comment by Alcaro** [comment]
I'm getting a slightly different error.
```
CC gfx/common/drm_common.c
In file included from gfx/common/drm_common.h:22:0,
                 from gfx/common/drm_common.c:19:
/usr/include/xf86drm.h:40:17: fatal error: drm.h: No such file or directory
 #include <drm.h>
                 ^
```

**Comment by Alcaro** [comment]
Should be fixed as of 008db1e.

**Comment by CanadianJeff** [comment]
closing because it compiles now
